polkadot/node/network/collator-protocol/src/validator_side/collation.rs

```diff
@@ -269,21 +302,20 @@ impl Collations {
 		match self.status {
 			// We don't need to fetch any other collation when we already have seconded one.
 			CollationStatus::Seconded => None,
```

Comment:
I wonder if this is still true with async backing. Shouldn't we accept `max_candidate_depth + 1` collations?

Reply:
the `self.status.back_to_waiting(relay_parent_mode);` above will reset the status to Waiting unless async backing is disabled. Only the comment is misleading